plugin/http/httptrace/httptrace_test.go

```diff
@@ -273,3 +275,38 @@ func serveHTTP(done chan struct{}, wait chan time.Time) string {
 	}()
 	return server.URL
 }
+
+func TestSpanNameFromURL(t *testing.T) {
+	tests := []struct {
+		prefix string
+		u      string
+		want   string
+	}{
+		{
+			prefix: "Sent",
+			u:      "http://localhost:80/hello?q=a",
+			want:   "Sent.localhost/hello",
+		},
+		{
+			prefix: "Recv",
+			u:      "https://localhost:443/a",
```

Comment:
Could you add an example with a non-standard port?

Reply:
Path will be represented as an attributed. Added a case for a non-standard port. PTAL.